data_collection/web/guides.rst

```diff
@@ -95,6 +100,163 @@ Tracking more sophisticated events might require attaching listeners to the DOM
+
+To track a category view, use :ref:`setEcommerceView<jtc-api-setEcommerceView>` method **before** tracking the page view, like this::
+
+    // set category to "Smartphones"
+    _paq.push(["setEcommerceView", false, false, "Smartphones"]);
+
+    // track page view
+    _paq.push(["trackPageView"]);
+
+The same method can be used for tracking product views. Again, it must be called **before** tracking a page view. Example::
+
+    // set product
+    _paq.push(["setEcommerceView",
+        "71253029",              // SKU (stock-keeping unit)
+        "SUPER Phone A40 White", // name
+        "Smartphones",           // category
+        1499.99                  // price
+    ]);
+
+    // track page view
+    _paq.push(["trackPageView"]);
+
+``category`` parameter of the :ref:`setEcommerceView<jtc-api-setEcommerceView>` method accepts not only string values, but also arrays of strings. This is useful for tracking products that belong to more than one category, or tracking pages that list products from multiple categories. ::
+
+    // set product to "SUPER Watch B20 Silver" belonging to 2 categories
+    _paq.push(["setEcommerceView", "00492710", "SUPER Watch B20 Silver", ["New offer", "Smartwatches"], 700.00]);
+
+    // track page view
+    _paq.push(["trackPageView"]);
+
+.. _guide_tracking_cart_updates:
+
+Tracking cart updates
+^^^^^^^^^^^^^^^^^^^^^
+
+Another type of e-commerce activity you can track is an update to a shopping cart. With it, we are able to measure how often visitors don't complete the ordering process and what products stay in abandoned carts.
+
+Tracking a cart update has two steps: registering items from the cart and sending them. The following example uses two methods - :ref:`addEcommerceItem<jtc-api-addEcommerceItem>` and :ref:`trackEcommerceCartUpdate<jtc-api-trackEcommerceCartUpdate>` - to achieve exactly that. ::
+
+    // visitor added one chocolate bar to an empty shopping cart
+
+    // register chocolate bar
+    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
+
+    // track cart update
+    _paq.push(["trackEcommerceCartUpdate", 6.00]);
+
+This code snippet sends a cart update event with a cart containing one item (SKU *candy-12837*, name *MEGA Milk Chocolate 200g*, category *Candy*, price *6.00*) and having total value of *6.00*.
+
+The list of registered items is stored only in memory. **Reloading the page will clear the list** and the previously registered items will have to be added again. ::
+
+    // visitor added one mango fruit to a shopping cart with one chocolate bar
+
+    // register previously added items
+    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
+
+    // register the new item
+    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
+
+    // track cart update
+    _paq.push(["trackEcommerceCartUpdate", 10.00]);
```

Comment:
Could you describe the parameters as the comment in the code the same way as you described it at the first occurrence of the `setEcommerceView` ?